app/controllers/aws_texts_controller.rb

```diff
@@ -4,5 +4,7 @@ def index
   end
 
   def show
+    @aws_texts_title = AwsText.find(params[:id]).title
+    @aws_texts_content = AwsText.find(params[:id]).content
```

Comment:
コントローラでタイトルと内容を分ける必要はありません。

```suggestion
    @aws_texts = AwsText.find(params[:id])
```

Reply:
わかりました！
修正しました！